package.json

```diff
@@ -165,5 +187,38 @@
       "__DEVELOPMENT__": true,
       "cozy": {}
     }
+  },
+  "eslintConfig": {
+    "extends": [
+      "standard",
+      "standard-react",
+      "prettier",
+      "prettier/react",
+      "prettier/standard"
+    ],
+    "plugins": [
+      "react",
+      "prettier",
+      "standard"
+    ],
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      }
+    },
+    "rules": {
+      "prettier/prettier": "error",
+      "react/prop-types": [2, {"skipUndeclared":true}],
+      "react/jsx-no-bind": 1
+    },
+    "env": {
+      "browser": true,
+      "jest": true
+    }
+  },
+  "prettier": {
+    "semi": false,
+    "singleQuote": true
   }
```

Comment:
IMHO it is better to use a file .eslintrc for this config, it will be easier to copy it to all our other apps and works better with `git diff` etc..

Reply:
It ain't eslint configuration, but prettier configuration.
I've written those rules inside `package.json` because I prefer to centralize things.
I am not a big fan of having multiple `.dotfiles` in the project's root.
As it is subjective, I can discuss on it:
https://www.loomio.org/p/UxBcK5nb/prettier-s-configuration-goes-to-prettierrc-or-in-package-json

Reply:
> It ain't eslint configuration, but prettier configuration.

Sorry I did not point the exact line, the eslint configuration is just above. For me there is one point that is not subjective : with a file, you can diff easily and copy without any problem. With a property in a JSON file, you cannot do that. In addition, the file can be read by multiple software, here it is coupled to webpack.